src/builtins.h

```diff
@@ -38,4 +38,14 @@ void AddBitcodeToModule(const BitcodeLib *lib, llvm::Module *module, SymbolTable
  */
 Symbol *CreateISPCSymbolForLLVMIntrinsic(llvm::Function *func, SymbolTable *symbolTable);
 
+#ifdef ISPC_XE_ENABLED
+std::string mangleMathOCLBuiltin(const llvm::Function &func);
+
+std::string manglePrintfOCLBuiltin(const llvm::Function &func);
+
+std::string mangleOCLBuiltin(const llvm::Function &func);
```

Comment:
You only need `mangleOCLBuiltin` and `mangleSPIRVBuiltin` as external methods. `mangleMathOCLBuiltin` and `manglePrintfOCLBuiltin` can be left as static and removed from header file.